MainDlg.cpp

```diff
@@ -17,16 +18,44 @@ MainDlg::MainDlg(CWnd* pParent /*=NULL*/)
 void MainDlg::DoDataExchange(CDataExchange* pDX)
 {
 	CDialog::DoDataExchange(pDX);
+	DDX_Control(pDX, IDC_LIST1, list_control);
 }
 
 BEGIN_MESSAGE_MAP(MainDlg, CDialog)
+	ON_BN_CLICKED(IDC_BUTTON1, AddItem)
+	ON_BN_CLICKED(IDC_BUTTON2, DeleteItem)
 END_MESSAGE_MAP()
 
+void MainDlg::AddItem() {
+	CString title;
+	CString year;
+
+	GetDlgItemText(IDC_EDIT1, title);
+	GetDlgItemText(IDC_EDIT2, year);
+
+	auto count = list_control.GetItemCount();
+
+	if (title && atoi(year)) {

```

Comment:
Što bi trebao provjeriti `if (title)` - title je string? 
Za provjeru je li string validna godina u C++-u ima puno boljih varijanti od `atoi`.